apis/iam/v1alpha1/policy.go

```diff
@@ -0,0 +1,353 @@
+package v1alpha1
+
+import xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+
+// Policy An Identity and Access Management (IAM) policy, which
+// specifies access
```

Comment:
nitpick:
```suggestion
// An Identity and Access Management (IAM) policy, which
// specifies access
```

Reply:
Need some comment starting with `Policy` to make linter happy.
Updates as "Policy is an Identity ..." to fix grammar.